### PR TITLE
PS-9305 8.4.0-1 release notes and release note tasks

### PR DESCRIPTION
**Pull request by patrickbirch**
On branch ps-9305
	new file:   docs/release-notes/8.4.0-1.md
	modified:   docs/release-notes/release-notes-index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release-notes/8.4.0-1.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+* [PS-9121](https://perconadev.atlassian.net/browse/PS-9121): MySQL exited when InnoDB failed to update a spatial index.
+
+* [PS-9151](https://perconadev.atlassian.net/browse/PS-9151): Percona server 8.0 build failed on CentOS 7 with `-DWITH_SSL=openssl11`.
+
+* [PS-9219](https://perconadev.atlassian.net/browse/PS-9219): While converting the charset collation in a table, MySQL converted the date and time data types columns in the `.ibd` file. However, the `collation_id` in the `.ibd` file did not align with that of the data dictionary.
+
+* [PS-9155](https://perconadev.atlassian.net/browse/PS-9155): The server exited during the execution of the complicated query with 9 CTEs.
+
+* [PS-9219](https://perconadev.atlassian.net/browse/PS-9219): While converting the charset collation in a table, MySQL converted the date and time data types columns in the `.ibd` file. However, the `collation_id` in the `.ibd` file did not align with that of the data dictionary.
```
Comment (percona-ysorokin) [inline docs/release-notes/8.4.0-1.md]: Duplicate item

File: docs/release-notes/8.4.0-1.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+* [PS-8963](https://perconadev.atlassian.net/browse/PS-8963): The `SEQUENCE_TABLE()` function is deprecated and may be removed in a future release. We recommend that you use `PERCONA_SEQUENCE_TABLE()` instead. To maintain compatibility with existing third-party software, `SEQUENCE_TABLE` is no longer a reserved term and can be used as a regular identifier. Find more information in [PERCONA_SEQUENCE_TABLE(n) function](../percona-sequence-table.md)
+
+## Packaging notes
+
+Percona Server for MySQL 8.0.37-29 supports Ubuntu 24.04.
+
+## Additional resources
+
+* Install [Percona Server for MySQL 8.3]
```
Comment (percona-ysorokin) [inline docs/release-notes/8.4.0-1.md]: Is this `8.3` intentional?

File: docs/release-notes/8.4.0-1.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+ # Percona Server for MySQL 8.4.0-1 (2024-08-)
+
+Get started with [Quickstart Guide for Percona Server for MySQL].
+
+[Percona Server for MySQL 8.4.0-1] includes all the features and bug fixes available in the
+MySQL 8.4 Community Edition in addition to enterprise-grade features developed by Percona.
+
+## Release highlights
+
+Improvements and bug fixes introduced by Oracle for MySQL 8.3 and included in Percona Server for MySQL are the following:
```
Comment (percona-ysorokin) [inline docs/release-notes/8.4.0-1.md]: Is `8.3` intentional here?

**Review by percona-ysorokin — CHANGES_REQUESTED** [review]

**Review by alina-derkach-oaza — APPROVED** [review]

File: docs/release-notes/8.4.0-1.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+* [Contribute to the documentation]
+
+* For [training], contact [Percona Training - Start learning now]
+
+[Quickstart Guide for Percona Server for MySQL]: ../quickstart-overview.md
+
+[Quickstart Guide for Percona Server for MySQL]: ../quickstart-overview.md
+[Percona Server for MySQL 8.4.0-1]: https://www.percona.com/software/mysql-database/percona-server
+[MySQL 8.4 Release Notes]: https://dev.mysql.com/doc/relnotes/mysql/8.3/en/news-8-4-0.html
```
Comment (percona-ysorokin) [inline docs/release-notes/8.4.0-1.md]: Wrong URL

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Pull request merged**